OrbitGl/PickingManager.h

```diff
@@ -56,11 +56,19 @@ struct PickingID {
     return id;
   }
   static Color GetColor(Type a_Type, uint32_t a_ID, BatcherId batcher_id = TIME_GRAPH) {
+    static_assert(sizeof(PickingID) == sizeof(Color),
+      "PickingId and Color must have the same size");
```

Comment:
Nit: I like having that static_assert. Ideally we also check whether the types are `std::is_trivially_copyable`.

Reply:
Good idea, I added this and found out that Color is not trivially copyable. To be on the safe side, I copy the values in an array first and then call the actual constructor of Color. 

Added static asserts where appropriate, I think (let me know if I missed something).

Reply:
Ahh good Point. I remember having that problem before. If I remember correctly Color is a gteVector4, which is not trivially copyable. At some point we should replace gteVector4 by something more appropriate, but for now this change looks good! Thanks.